src/main/java/org/mskcc/cmo/messaging/model/PatientMetadata.java

```diff
@@ -0,0 +1,78 @@
+package org.mskcc.cmo.messaging.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+import org.neo4j.ogm.id.UuidStrategy;
+import org.neo4j.ogm.typeconversion.UuidStringConverter;
+
+/**
+ *
+ * @author ochoaa
+ */
+@NodeEntity
+public class PatientMetadata implements Serializable {
+    @Id @GeneratedValue(strategy = UuidStrategy.class)
+    @Convert(UuidStringConverter.class)
+    private UUID metaDbUuid;
+    private String investigatorPatientId;
+    @Relationship(type="PX_TO_SP", direction=Relationship.OUTGOING)
+    private List<SampleMetadataEntity> sampleMetadataList;
```

Comment:
Curious why sample metadata is required here.

Reply:
to define the relationship between the patient metadata node and sample metadata nodes it's connected with